pkg/ipamplugin/ipam_plugin.go

```diff
@@ -272,19 +272,19 @@ func cmdAdd(args *skel.CmdArgs) error {
 			defer unlock()
 			return calicoClient.IPAM().AutoAssign(ctx, assignArgs)
 		}
-		assignedV4, assignedV6, err := autoAssignWithLock(calicoClient, ctx, assignArgs)
-		logger.Infof("Calico CNI IPAM assigned addresses IPv4=%v IPv6=%v", assignedV4, assignedV6)
```

Comment:
I think it might be nice to keep this

Reply:
I just moved it down to after the err check because v4Assignments/v6Assignments can be nil in this case, but I see it doesn't make sense because I later changed the logging to print the whole struct instead of just the IPs slice inside it. Will move it back, thanks!